ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -108,6 +115,15 @@ export class FrontstageDef {
   public get bottomCenter(): ZoneDef | undefined { return this._bottomCenter; }
   public get bottomRight(): ZoneDef | undefined { return this._bottomRight; }
```

Comment:
@raplemie we still have some properties that need to be deprecated, let's do that in a separate PR..